src/qtbase/QColor.js

```diff
@@ -1,32 +1,78 @@
-// TODO complete implementation (with attributes `r`,`g` and `b`).
-
 class QColor {
   constructor(val) {
+    this.r = this.g = this.b = 0;
```

Comment:
Because QML `color` type actually does have `r`, `g`, `b`, and `a` properties, and those are in 0-1 range:

```qml
import QtQuick 2.0

Item {
    property color bar: 'green'
    Component.onCompleted: {
      console.log(bar.r, bar.g, bar.b, bar.a, bar)
    }
}
```